renderdoc/driver/gl/gl_replay_egl.cpp

```diff
@@ -0,0 +1,100 @@
+/******************************************************************************
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015-2017 Baldur Karlsson
+ * Copyright (c) 2014 Crytek
```

Comment:
For new files please just use `Copyright (c) 201X Baldur Karlsson` if you copy-paste from elsewhere.